### PR TITLE
feat: allow viz height and width parameters

### PR DESCRIPTION
**Pull request by jorwoods**
Add support to PDFRequestOptions for viz height and width.

### CODE REVIEW

File: tableauserverclient/server/request_options.py [hunk 1]
```diff
@@ -287,6 +309,15 @@ def get_query_params(self):
         if self.max_age != -1:
             params["maxAge"] = self.max_age
 
+        if (self.viz_height is None) ^ (self.viz_width is None):
```
Comment (jacalata) [inline tableauserverclient/server/request_options.py:312]: I think this operator is unusual enough that a comment would be helpful

**Review by jacalata — APPROVED** [review]
Thanks!

File: tableauserverclient/server/request_options.py (hunk 1 above)
Comment (jorwoods) [reply tableauserverclient/server/request_options.py:313]: Added comment.

**Comment by jorwoods** [comment]
Resolves #1324

**Pull request merged**